.docusaurus/docusaurus-plugin-content-docs/default/version-current-metadata-prop-751.json

```diff
@@ -40,235 +40,235 @@
-                "docId": "yah-cli/Commands/yah-push"
+                "label": "bb push",
+                "href": "/docs/bb-cli/Commands/bb-push",
+                "docId": "bb-cli/Commands/bb-push"
               },
               {
                 "type": "link",
-                "label": "yah push-config",
-                "href": "/docs/yah-cli/Commands/yah-push-config",
-                "docId": "yah-cli/Commands/yah-push-config"
+                "label": "bb push-config",
+                "href": "/docs/bb-cli/Commands/bb-push-config",
+                "docId": "bb-cli/Commands/bb-push-config"
               },
               {
                 "type": "link",
-                "label": "yah exec",
-                "href": "/docs/yah-cli/Commands/yah-exec",
-                "docId": "yah-cli/Commands/yah-exec"
+                "label": "bb exec",
+                "href": "/docs/bb-cli/Commands/bb-exec",
+                "docId": "bb-cli/Commands/bb-exec"
               },
               {
                 "type": "link",
-                "label": "yah ls",
-                "href": "/docs/yah-cli/Commands/yah-ls",
-                "docId": "yah-cli/Commands/yah-ls"
+                "label": "bb ls",
+                "href": "/docs/bb-cli/Commands/bb-ls",
+                "docId": "bb-cli/Commands/bb-ls"
               },
               {
                 "type": "link",
-                "label": "yah sync",
-                "href": "/docs/yah-cli/Commands/yah-sync",
-                "docId": "yah-cli/Commands/yah-sync"
+                "label": "bb sync",
+                "href": "/docs/bb-cli/Commands/bb-sync",
+                "docId": "bb-cli/Commands/bb-sync"
               },
               {
                 "type": "link",
-                "label": "yah pull",
-                "href": "/docs/yah-cli/Commands/yah-pull",
-                "docId": "yah-cli/Commands/yah-pull"
+                "label": "bb pull",
+                "href": "/docs/bb-cli/Commands/bb-pull",
+                "docId": "bb-cli/Commands/bb-pull"
               }
             ]
           }
         ]
       },
       {
         "type": "category",
-        "label": "Blox Store",
+        "label": "Appblock SDK",
         "collapsible": true,
         "collapsed": true,
         "items": [
           {
             "type": "link",
-            "label": "Blocks Store Walkthrough",
-            "href": "/docs/blocks-store/walkthrough",
-            "docId": "blocks-store/walkthrough"
+            "label": "@appblocks/js-sdk",
+            "href": "/docs/SDK/js-sdk",
+            "docId": "SDK/js-sdk"
+          },
+          {
+            "type": "link",
+            "label": "@appblocks/node-sdk",
+            "href": "/docs/SDK/node-sdk",
+            "docId": "SDK/node-sdk"
           }
         ]
       },
       {
         "type": "category",
-        "label": "Yah SDKs",
+        "label": "Blox Store",
```

Comment:
we don't use blox any more its blocks right?

Reply:
Yes, I will update it.